packages/valory/skills/termination_abci/behaviours.py

```diff
@@ -622,32 +510,6 @@ def async_act(self) -> Generator:
         yield
 
 
-class PostTerminationTxAbciApp(AbciApp):
-    """The abci app running after the multisig transaction has been settled."""
-
-    initial_round_cls = TerminationRound
-    # the following is not needed, it is added to satisfy the round check
-    # the TerminationRound when run it terminates the agent, so nothing can come after it
-    transition_function = {TerminationRound: {Event.TERMINATE: TerminationRound}}
-
-
-termination_transition_function = {
-    FinishedTransactionSubmissionRound: PostTerminationTxAbciApp.initial_round_cls,
-    FailedRound: TransactionSubmissionAbciApp.initial_round_cls,
-}
-
-TerminationAbciApp = chain(
-    (
-        TransactionSubmissionAbciApp,
-        PostTerminationTxAbciApp,
-    ),
-    termination_transition_function,
-)
-TerminationAbciApp.transition_function[BackgroundRound] = {
-    Event.TERMINATE: TransactionSubmissionAbciApp.initial_round_cls,
-}
-
-
 class TerminationAbciBehaviours(AbstractRoundBehaviour):
```

Comment:
It's confusing that this is already a "chained" behaviour. Have a look at https://github.com/valory-xyz/apy-oracle/blob/main/packages/valory/skills/apy_estimation_abci/behaviours.py#L1562, we just define the standalone ABCI app, here it would be two: BackgroundAbciBehaviours & TerminationAbciBehaviours.

Reply:
I would say that we don't need the `BackgroundAbciBehaviours` here. It's a single behavior only.